ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityBlock/activityBlock.component.html

```diff
@@ -10,6 +10,7 @@
                                           [validationRequested]="validationRequested"
                                           [studyGuid]="studyGuid"
                                           [parentActivityInstanceGuid]="parentActivityInstanceGuid"
+                                          (componentBusy)="embeddedComponentBusy.emit($event)"
```

Comment:
I used the same `embeddedComponentBusy` emitter because it serves for the same goal - to inform on the top level (`activityForm/activity.component.ts`) that a nested component is busy.